apps/meteor/app/api/server/helpers/getUserInfo.ts

```diff
@@ -25,13 +25,49 @@ const getUserPreferences = async (me: IUser): Promise<Record<string, unknown>> =
 	return accumulator;
 };
 
+/**
+ * Returns the user's calendar settings based on their email domain and the configured mapping.
+ * If the email is not provided or the domain is not found in the mapping,
+ * it returns the default Outlook calendar settings.
+ * @param email - The user's email object, which may contain the address and verification status.
+ * @returns The calendar settings for the user, including Outlook calendar settings if enabled.
+ */
+const getUserCalendar = (email: false | IUserEmail | undefined): IUserCalendar => {
+	const calendarSettings: IUserCalendar = {};
+
+	const outlook = {
+		Enabled: settings.get<boolean>('Outlook_Calendar_Enabled'),
+		Exchange_Url: settings.get<string>('Outlook_Calendar_Exchange_Url'),
+		Outlook_Url: settings.get<string>('Outlook_Calendar_Outlook_Url'),
```

Comment:
Could this be `enabled`, `exchangeUrl` and `outlookUrl`?

Reply:
I considered that, but it's better to keep consistency with the setting names.